docs/user_guide/dashboard_api.rst

```diff
@@ -3,57 +3,67 @@
 #########################
 Dashboard in NVIDIA FLARE
 #########################
+
 As mentioned in :ref:`provisioning`, the NVIDIA FLARE system requires a set of startup kits
-which include the private keys and certificates (signed by the root CA) in order to communicate with one another.
-The new :ref:`nvflare_dashboard_ui` in NVIDIA FLARE provides a simple way to collect information about clients and users from different organizations,
-as well as to generate those startup kits for users to download.
+that include private keys and certificates (signed by the root CA) for secure communication between components.
+The :ref:`nvflare_dashboard_ui` in NVIDIA FLARE provides a streamlined way to collect information about clients and users from different organizations,
+and to generate startup kits for users to download.
 
-Most of the details about provisioning can be found in :ref:`provisioning`.  In this section, we focus on the user interaction with Dashboard and its backend APIs.
+For detailed information about provisioning, refer to :ref:`provisioning`. This section focuses on user interaction with the Dashboard and its backend APIs.
 
 .. include:: nvflare_cli/dashboard_command.rst
 
 **********************************
-NVIDIA FLARE Dashboard backend APIs
+NVIDIA FLARE Dashboard Backend APIs
 **********************************
 
 Architecture
 ============
 
-The Dashboard backend APIs follows the RESTful concept. It defines four resources: Project, Organizations, Client, and User.  There is one and only one Project.
-The Project includes information about server(s) and overseer (if in HA mode).  Clients are defined for NVIDIA FLARE clients and Users for NVIDIA FLARE admin console.
-Organizations is a GET-only operation that returns a list of currently registered organizations.
+The Dashboard backend APIs follow RESTful principles and define four main resources: Project, Organizations, Client, and User. There is exactly one Project per deployment,
+which includes information about the server (the overseer and additional server for HA mode were removed in NVIDIA FLARE 2.6). Clients represent NVIDIA FLARE client instances, while Users represent NVIDIA FLARE admin console users.
```

Comment:
maybe we don't need to say "the overseer and additional server for HA mode were removed in NVIDIA FLARE 2.6" here? since we already mentioned it in the release notes?